engine/schema/src/main/resources/META-INF/db/schema-41900to42000.sql

```diff
@@ -79,3 +79,7 @@ CREATE TABLE IF NOT EXISTS `cloud_usage`.`quota_email_configuration`(
     PRIMARY KEY (`account_id`, `email_template_id`),
     CONSTRAINT `FK_quota_email_configuration_account_id` FOREIGN KEY (`account_id`) REFERENCES `cloud_usage`.`quota_account`(`account_id`),
     CONSTRAINT `FK_quota_email_configuration_email_template_id` FOREIGN KEY (`email_template_id`) REFERENCES `cloud_usage`.`quota_email_templates`(`id`));
+
+-- Fields related to Snapshot Extraction
+ALTER TABLE `cloud`.`snapshot_store_ref` ADD COLUMN `download_url` varchar(2048) NULL;
+ALTER TABLE `cloud`.`snapshot_store_ref` ADD COLUMN `download_url_created` datetime NULL;
```

Comment:
Could use the procedure `IDEMPOTENT_ADD_COLUMN` here.

Reply:
could == should ;)

Reply:
True. Fixed.